lib/runtime/wasmer/imports_test.go

```diff
@@ -56,6 +57,17 @@ func TestMain(m *testing.M) {
 	os.Exit(code)
 }
 
+func Test_ext_offchain_timestamp_version_1(t *testing.T) {
+	inst := NewTestInstance(t, runtime.HOST_API_TEST_RUNTIME)
+
+	ret, err := inst.Exec("ext_offchain_timestamp_version_1", nil)
+	require.NoError(t, err)
```

Comment:
just curious, was the `HOST_API_TEST_RUNTIME` updated by you? generally, exported wasm functions that call imported (ext_) functions are prefixed with `rtm_`

Reply:
I did not. I tried with `rtm_` but it failed to find the fn. Hence I removed it. I mentioned this as an issue in the initial pull request. Also, due to this, I did not add a unit test for 2nd function. I will give it another go!